src/Twilio.Api.Silverlight/Calls.Async.cs

```diff
@@ -26,7 +26,7 @@ public virtual void ListCalls(Action<CallResult> callback)
 		/// </summary>
 		/// <param name="options">List filter options. If an property is set the list will be filtered by that value.</param>
 		/// <param name="callback">Method to call upon successful completion</param>
-        public virtual void ListCalls(CallListRequest options, Action<CallResult> callback)
+		public virtual void ListCalls(CallListRequest options, Action<CallResult> callback)
 		{
 			var request = new RestRequest();
 			request.Resource = "Accounts/{AccountSid}/Calls.json";
```

Comment:
I'm not allowed to comment below, but can you remove the commented lines in the method below?